### ISSUE TITLE
Ultraviolet light exposure, strong

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9976]
**Preferred term label:**
Ultraviolet light exposure, strong

**Synonyms**
UV; outdoor activities, living near equator, radiation

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pterygium

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001092
- label: Strong ultraviolet light exposure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: History of exposure to ultraviolet light, such as may be encountered with outdoor activities, living near equator, radiation.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9976
- synonyms:
